packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -78,6 +79,13 @@ function isReadonlyArray(arg: any): arg is ReadonlyArray<any> {
   return Array.isArray(arg)
 }
 
+if (typeof TARGET_ENGINE_TYPE !== 'undefined' && TARGET_ENGINE_TYPE === 'all') {
+  console.warn('imports from "@prisma/client/runtime" are deprecated.')
+  console.warn(
```

Comment:
How does that work for default-index.d.ts?

Reply:
Ah, forgot about those. I guess right now it will produce a warning and we have no good way to pick up right runtime before client is generated. Do you think it would make sense to remove a warning for now then?

Reply:
Slightly related: on the type emission, how does it work? Are we emitting double the amount of types (1 for each runtime)? (runtime, edge, binary, library)?

Reply:
We are emitting single `.d.ts` file for default runtime, then specific ones just genereate `export * from './index'` definition.

Reply:
Nice thanks